draft-ietf-aipref-attach.md

```diff
@@ -149,24 +149,20 @@ Embedding preferences is expected to be an effective means
 of associating preferences with content,
 because it ensures that metadata is always associated with content.
 
-The main challenge with embedding is that
-a different method is needed for each content type.
-That is,
-a different means of conveying preferences
-needs to be defined for each audio, documents, images, video,
-or other content format.
+Each content type - whether audio, documents, images, video, or other - 
+embeds metadata in different ways, however, in most cases it is possible to 
+embed the same metadata, just in different places. This is, for example, how XMP (ISO 16684) 
```

Comment:
We'll need a citation for that, even if it is just normative.  Do you have a link?

Reply:
The official ISO version is at https://www.iso.org/standard/75163.html.  The Adobe version is at https://github.com/adobe/XMP-Toolkit-SDK/blob/main/docs/XMPSpecificationPart1.pdf